src/WorkerPool.js

```diff
@@ -186,7 +187,11 @@ class PoolWorker {
     const err = new Error(obj.message);
     err.message = obj.message;
     if (obj.stack) {
-      err.stack = `${obj.stack}\n\tfrom thread-loader (worker ${this.id})\n${err.stack}`;
+      let additionalStack = '';
```

Comment:
```js
err.stack = `${obj.stack}\n\nThread Loader (Worker ${this.id})`

if (this.options.stack) {
   // Limit the err.stack manually in case stackTraces are to verbose
   const limit = (stack) => typeof options.stack === 'number' 
     ? Error.stackTraceLimit(options.stack)
     : stack

   err.stack += `\n\n${limit(err.stack)}`
} else {
   // Only the err.message
   const message = (stack) => stack.split('\n')[0]

   err.stack += `\n\n${message(err.stack)}`
}
```

Reply:
I like this idea. the stacktraceLimit is not a function but fairly. I will change the implementation.